src/main.rs

```diff
@@ -17,14 +17,15 @@ use uuid::Uuid;
 
 #[tokio::main]
 async fn main() {
-    let tracker = Snowplow::create_tracker("ns", "app_id", "http://localhost:9090");
+    let tracker = Snowplow::create_tracker("ns", "app_id", "http://localhost:9090", None);
 
     let self_desc_event_id = tracker
         .track(
             SelfDescribingEvent {
                 schema: "iglu:com.snowplowanalytics.snowplow/screen_view/jsonschema/1-0-0"
                     .to_string(),
                 data: json!({"name": "test", "id": "something else"}),
+                subject: None,
```

Comment:
Nit: it's probably better to use the builder pattern in examples now. Also gives us more flexibility in the future to avoid breaking user code while adding properties to the structs.